tests/Integration/Behaviour/Features/Scenario/Product/SpecificPrice/edit_specific_price.feature

```diff
@@ -0,0 +1,291 @@
+    Then I should get error that shop was not found
+    When I edit specific price "price1" with following details:
+      | currency | jen |
+    Then I should get error that currency was not found
+    When I edit specific price "price1" with following details:
+      | country | Simsalabim |
+    Then I should get error that country was not found
+    When I edit specific price "price1" with following details:
+      | group | pirates |
+    Then I should get error that group was not found
+    When I edit specific price "price1" with following details:
+      | customer | Rick - the pickle |
+    Then I should get error that customer was not found
+    And specific price price1 should have following details:
+      | specific price detail | value  |
+      | reduction type        | amount |
+      | reduction value       | 12.56  |
+      | includes tax          | true   |
+      | price                 | 45.78  |
+      | from quantity         | 1      |
+      | from                  |        |
+      | to                    |        |
+      | shop                  |        |
+      | currency              |        |
+      | country               |        |
+      | group                 |        |
+      | customer              |        |
+
+  Scenario: I edit specific price dates
+    Given specific price price1 should have following details:
+      | specific price detail | value  |
+      | reduction type        | amount |
+      | reduction value       | 12.56  |
+      | includes tax          | true   |
+      | price                 | 45.78  |
+      | from quantity         | 1      |
+      | from                  |        |
+      | to                    |        |
+    When I edit specific price "price1" with following details:
+      | from | 2021-01-01 10:00:00 |
+      | to   | 2021-01-01 11:00:00 |
+    Then specific price price1 should have following details:
+      | specific price detail | value               |
+      | reduction type        | amount              |
+      | reduction value       | 12.56               |
+      | includes tax          | true                |
+      | price                 | 45.78               |
+      | from quantity         | 1                   |
+      | from                  | 2021-01-01 10:00:00 |
+      | to                    | 2021-01-01 11:00:00 |
+    When I edit specific price "price1" with following details:
+      | from | 0000-00-00 00:00:00 |
+      | to   | 0000-00-00 00:00:00 |
+    Then specific price price1 should have following details:
+      | specific price detail | value               |
+      | reduction type        | amount              |
+      | reduction value       | 12.56               |
+      | includes tax          | true                |
+      | price                 | 45.78               |
+      | from quantity         | 1                   |
+      | from                  | 0000-00-00 00:00:00 |
+      | to                    | 0000-00-00 00:00:00 |
```

Comment:
I wonder here, is there a difference between:
`| to | |`
and
`| to | 0000-00-00 00:00:00 |`

I wonder this because I want to be sure we can "unassign" a date in from and to, and the specific price can rollback to an empty state like it was in L210 This is important so that the merchant can "undo" a modification he made by mistake

Which is why I'd like you to add a test that ensures:
```
Then specific price price1 should have following details:
      | from                  |        |
      | to                    |        |
```

Reply:
If both statements are not equivalent (because we can't set back `null` in DB or something like that) then I believe there is a small problem to be fixed

Reply:
check full sc `I edit specific price dates`. It uses both cases.
The empty case (`from | | ` and `to | | `) is equivalent to values not being changed at all (null) so the handler ignores them and leaves previous ones.
The case with 0000-00-00 is equivalent to value being reset